### PR TITLE
Update sc.php

### PR DESCRIPTION
**Pull request by OlegBalakhnin**
Fix
Warning: A non-numeric value encountered in /usr/local/www/apache24/data/billing/api/libs/api.fundsflow.php on line 230

### CODE REVIEW

**Comment by nightflyza** [comment]
> Warning: A non-numeric value encountered in /usr/local/www/apache24/data/billing/api/libs/api.fundsflow.php on line 230

what?

**Comment by OlegBalakhnin** [comment]
`zb_CashAdd($userLogin, **'-'** . $creditFee, 'add', $creditCashType, 'SCFEE');`
writes a non-numeric value to the base where a numeric value is expected. Leads to warnings in addcach and pl _fundsflow modules. 
Should be replaced by
`zb_CashAdd($userLogin, **'0'** . $creditFee, 'add', $creditCashType, 'SCFEE');` 
or
`zb_CashAdd($userLogin, **'-0'** . $creditFee, 'add', $creditCashType, 'SCFEE');`
as in the personal account credit module.

**Comment by nightflyza** [comment]
>>writes a non-numeric value to the base where a numeric value is expected.

Writing non numeric cash(!) values is unexpected in normal world.

>>Leads to warnings in addcach and pl _fundsflow modules.

Read the changelog for few last month. You trying to "fix" already non existent issue.

>> zb_CashAdd($userLogin, **'-0'** . $creditFee, 'add', $creditCashType, 'SCFEE');

I wish I never saw it. 
Concatenate -0 as a string(?) to a known wrong value is solultion? Seriously? =)

File: modules/remoteapi/sc.php [hunk 1]
```diff
@@ -12,9 +12,9 @@
         $billing->setcredit($userLogin, $newCredit);
         $billing->setcreditexpire($userLogin, $creditExpire);
         //charging some money
-        zb_CashAdd($userLogin, '-' . $creditFee, 'add', $creditCashType, 'SCFEE');
+        zb_CashAdd($userLogin, '-0' . $creditFee, 'add', $creditCashType, 'SCFEE');
```
Comment (nightflyza) [inline modules/remoteapi/sc.php:15]: photo_2023-01-27_11-02-08

Comment (OlegBalakhnin) [reply modules/remoteapi/sc.php:15]: I'm sorry. thank you for your time.

**Pull request closed without merging**